### ISSUE TITLE
Task of error handler with Koa 2 cannot pass

### ISSUE THREAD

**Issue by xie-qianyue** [koajs/kick-off-koa#36]
In the task of error handler, the response cannot be set with `ctx.body = 'internal server error'`. Here is my error handler :  
```
function errorHandler(ctx, next) {
    try {
       return next();
    }
    catch(err) {
      ctx.status = err.status || 500;     
      ctx.body = 'internal server error';   
      // emit can't help
      // ctx.app.emit('error', err, ctx);      
    }
}
```
When I verify my codes, the third test always fails :  
3.  ACTUAL:    "responds \"Internal Server Error\" when requesting `/error`"
3.  EXPECTED:  "responds \"internal server error\" when requesting `/error`"

**Comment by dead-horse** [comment]
this project is only test in koa@1 now.  :(

**Comment by xie-qianyue** [comment]
@dead-horse Thanks for your reply. I've created a pull request which modifies the README to figure out that.

**Comment by scorbiclife** [comment]
Two things of note, possibly spoilers (click to expand)

<details>

<summary>About the solution criteria</summary>

It seems a passing solution only needs a matching response body, not the response status.
I guess the author intended a response 200 with different message bodies,
while some readers (including myself) think the status code has to match the route.
(So that `/500` returns `HTTP 500` etc.)

</details>

<details>

<summary>About setting the response body with HTTP status >= 500</summary>

As per the [http-errors docs](https://github.com/jshttp/http-errors#error-properties),
it seems one needs to set `err.expose = true` in order to set a custom response body.
That certainly doesn't seem like the author's intention, but is one way to solve the problem.

</details>

To summarize I think elaborating the solution criteria might be a good idea.
